metricbeat/module/system/process/process.go

```diff
@@ -0,0 +1,114 @@
+/*
+
+An example event looks as following:
+
+
+    {
+      "@timestamp": "2016-04-26T19:24:19.108Z",
+      "beat": {
+        "hostname": "ruflin",
+        "name": "ruflin"
+      },
+      "metricset": "process",
+      "module": "system",
+      "rtt": 20982,
+      "system-process": {
```

Comment:
@jpountz We actually decided to go without nested documents and have multiple documents instead. This means instead of having an array of documents, we send each document of this array separately to elasticsearch and make it then possible to "correlate" the documents based on timestamp and a unique identifier. As far as I understood, that is also more or less how lucene stores the nested documents?